script/openqa-clone-custom-git-refspec

```diff
@@ -55,6 +58,16 @@ extract_urls_from_pr() {
     fi
 }
 
+# Get the Github Token from a config file
+# located at ~/.config/openqa/verification.conf
+get_token_from_file() {
+  local config_file="$HOME/.config/openqa/verification.conf"
```

Comment:
See above, also provide an example of the format in the help file.

Reply:
fixed 👍🏼